### DISCUSSION TITLE
CVE for security vulnerability in intelowl

### DISCUSSION THREAD

**Discussion by spoiicy** [intelowlproject/IntelOwl#2292] [General]
I want to know if any CVE is assigned on finding any security vulnerabilities in IntelOwl. 
Would really appreciate any response from the team.

**Comment by mlodic** [comment]
hey, who am I talking to and which is your specific need?

**Reply by spoiicy** [reply]
I am a security researcher and developer who would like to contribute to this project in coming future. I was curious that if I do security testing for IntelOwl, will any CVE is assigned/given to valid vulnerability?

**Reply by mlodic** [reply]
if you report it responsibly, we can ask for a CVE. In the cases where we found vulns, we didn't ask CVEs yet cause there were really of low importance but we can start to do it once we find important issues: https://github.com/intelowlproject/IntelOwl/security/advisories

**Reply by mlodic** [reply]
and yes, you are welcome to test the application, feel free to ask for anything that can help in case

**Reply by spoiicy** [reply]
Thanks a lot @mlodic. I mean yah I'll be contributing to the project from a security standpoint as well adding new features. Looking forward to the v6 release as well XD.
